src/lib/Handset/PPMHandset.cpp

```diff
@@ -6,6 +6,7 @@
 #include "common.h"
 #include "crsf_protocol.h"
 #include "logging.h"
+#include "rmtallocator.h"
 
 #include <driver/rmt.h>
```

Comment:
Not required as `rmtallocator.h` includes rmt.h. I think we can safely say that including `rmtallocater.h` includes all the requirements for working with the RMT module.

Reply:
agreed